.asf.yaml

```diff
@@ -24,8 +24,16 @@ github:
     squash:  true
     merge:   false
     rebase:  false
+
 notifications:
   commits:      commits@ratis.apache.org
   issues:       issues@ratis.apache.org
   pullrequests: issues@ratis.apache.org
   jira_options: link label worklog
+
+staging:
+  profile: ~
+  whoami:  asf-staging
```

Comment:
Ratis repo currently doesn't have this branch.  May that cause any errors?  Also, what is the suggested workflow for updating the staging branch, then publishing the real site?

Reply:
Good points.  Let's remove it and skip staging for the moment.